pkg/frontend/assets/flowhouse.js

```diff
@@ -142,27 +142,58 @@ function drawChart() {
 }
 
 function renderChart(rdata) {
-  pres = Papa.parse(rdata.trim())
+  pres = Papa.parse(rdata.trim());
+
+  var filtered = [pres.data[0]];
+  for (var i = 1; i < pres.data.length; i++) {
+    var row = pres.data[i];
+    var hasNonZero = row.slice(1).some(function(val) {
+      var num = parseFloat((val || '').trim());
+      return !isNaN(num) && num !== 0;
+    });
+    if (hasNonZero) {
+      filtered.push(row);
+    }
+  }
 
   var data = [];
-  for (var i = 0; i < pres.data.length; i++) {
-    for (var j = 0; j < pres.data[i].length; j++) {
-      if (j == 0) {
-        data[i] = [];
-      }
-      x = pres.data[i][j];
-      if (i != 0) {
-        if (j != 0) {
-          x = parseInt(x)
-        }
+  for (var i = 0; i < filtered.length; i++) {
+    data[i] = [];
+    for (var j = 0; j < filtered[i].length; j++) {
+      var x = filtered[i][j];
+      if (i !== 0 && j !== 0) {
+        x = parseFloat((x || '').trim());
+        if (isNaN(x)) x = 0;
```

Comment:
[sand] style suggestions
```suggestion
  for (const [i, row] of filtered.entries()) {
    data[i] = [];
    for (const [j, col] of row.entries()) {
      let x = col;
      if (i !== 0 && j !== 0) {
        x = parseFloat((col || '').trim());
        if (isNaN(x)) {
          x = 0;
        }
```

Reply:
![image](https://github.com/user-attachments/assets/09d2ad5a-a7a9-409d-9f50-ce9ed0543985)
 breaks the time ranges